catalog/security/catalog-security-auditplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -14,8 +14,8 @@
 -->
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0">
 
-    <bean id="auditPlugin" class="ddf.catalog.security.audit.RetrieveResourceAuditPlugin"/>
+    <bean id="auditPlugin" class="ddf.catalog.security.audit.AuditPlugin"/>
 
-    <service ref="auditPlugin" auto-export="interfaces"/>
+    <service ref="auditPlugin" auto-export="interfaces" ranking="0"/>
```

Comment:
Should this have a high ranking? I am just thinking that if a plugin like the IngestPlugin (rank=100) would run first and could reject the request. Then this plugin would never get run and we wouldn't log the fact that someone tried to ingest.